apko/defs.bzl

```diff
@@ -0,0 +1,60 @@
+"Public API re-exports"
+
+URL="https%3A%2F%2Fdl-cdn.alpinelinux.org%2Falpine%2Fedge%2Fmain"
+
+def _impl(ctx):
```

Comment:
shouldn't an `impl` function like this go under the private folder?

Reply:
oh yes, this is not done yet.

Reply:
fixed